package.json

```diff
@@ -17,19 +17,20 @@
     "url": "https://github.com/chanzuckerberg/edu-design-system/issues"
   },
   "files": [
-    "/lib"
+    "/lib",
+    "tailwind.config.js"
   ],
   "scripts": {
     "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn build:styles && yarn copy-to-lib",
     "build:clean": "rm -rf lib/",
-    "build:tokens": "rm -rf src/tokens-dist/ && node ./style-dictionary.config.js",
+    "build:tokens": "rm -rf src/tokens-dist/ && node ./style-dictionary.config.js && yarn copy-tokens-to-lib",
```

Comment:
to extend the tailwind.config.js file in Traject, it needs to reference `lib/tokens` from the built output. this makes sure the folder gets copied over as part of token building